test/fixtures/roles.sql

```diff
@@ -7,46 +7,50 @@ CREATE ROLE postgrest_test_author;
 GRANT postgrest_test_anonymous, postgrest_test_default_role, postgrest_test_author TO :USER;
 
 -- reloadable config options for io tests
-ALTER ROLE postgrest_test_authenticator SET pgrst."jwt-aud" = 'https://example.org';
-ALTER ROLE postgrest_test_authenticator SET pgrst."openapi-server-proxy-uri" = 'https://example.org/api';
-ALTER ROLE postgrest_test_authenticator SET pgrst."raw-media-types" = 'application/vnd.pgrst.db-config';
-ALTER ROLE postgrest_test_authenticator SET pgrst."jwt-secret" = 'REALLYREALLYREALLYREALLYVERYSAFE';
-ALTER ROLE postgrest_test_authenticator SET pgrst."jwt-secret-is-base64" = 'true';
-ALTER ROLE postgrest_test_authenticator SET pgrst."jwt-role-claim-key" = '."a"."role"';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-tx-end" = 'commit-allow-override';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-schemas" = 'test, tenant1, tenant2';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-root-spec" = 'root';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-prepared-statements" = 'false';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-pre-request" = 'test.custom_headers';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-max-rows" = '1000';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-extra-search-path" = 'public, extensions';
+ALTER ROLE postgrest_test_authenticator SET pgrst."jwt_aud" = 'https://example.org';
+ALTER ROLE postgrest_test_authenticator SET pgrst."openapi_server_proxy_uri" = 'https://example.org/api';
+ALTER ROLE postgrest_test_authenticator SET pgrst."raw_media_types" = 'application/vnd.pgrst.db-config';
+ALTER ROLE postgrest_test_authenticator SET pgrst."jwt_secret" = 'REALLYREALLYREALLYREALLYVERYSAFE';
+ALTER ROLE postgrest_test_authenticator SET pgrst."jwt_secret_is_base64" = 'true';
+ALTER ROLE postgrest_test_authenticator SET pgrst."jwt_role_claim_key" = '."a"."role"';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_tx_end" = 'commit-allow-override';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_schemas" = 'test, tenant1, tenant2';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_root_spec" = 'root';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_prepared_statements" = 'false';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_pre_request" = 'test.custom_headers';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_max_rows" = '1000';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_extra_search_path" = 'public, extensions';
+
+-- override with database specific setting
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_max_rows" = '1111';
+ALTER ROLE postgrest_test_authenticator IN DATABASE postgres SET pgrst."db_extra_search_path" = 'public, extensions, private';
```

Comment:
```suggestion
ALTER ROLE postgrest_test_authenticator IN DATABASE :DBNAME SET pgrst."db_extra_search_path" = 'public, extensions, private';
```

I think you can use `:DBNAME` here, because the sql file will be loaded by `psql`. This way the database name is not hardcoded here.